.github/workflows/test-install-plugins.yml

```
@@ -0,0 +1,182 @@
+name: Test Installing Plugins
+
+# Test Top 50 asdf plugins install correctly and the most common can print their
+# current version
+
+on:
+  push:
+    tags: ["v*"]
+  schedule:
+    # run at midnight on sunday (utc?)
+    - cron: 0 0 * * 0
+  # TODO: Remove `on: pull_request` once CI is verified to run correctly
+  pull_request:
+    branches: ["main"]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-linux:
+    name: Build linux
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: x86_64-unknown-linux-gnu
+          override: true
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: "x86_64-unknown-linux-gnu"
+      - uses: actions-rs/cargo@v1
+        with:
+          use-cross: true
+          command: build
+          args: --release --all-features --target x86_64-unknown-linux-gnu
+      - run: scripts/build-tarball.sh x86_64-unknown-linux-gnu
```

the syntax for this changed, so you'll want to do this:

```suggestion
      - run: scripts/build-tarball.sh rtx --release --target x86_64-unknown-linux-gnu
```